ArduCopter/mode.h

```diff
@@ -1853,6 +1854,9 @@ class ModeTurtle : public Mode {
     float motors_output;
     Vector2f motors_input;
     uint32_t last_throttle_warning_output_ms;
+
+    HAL_Semaphore msem;
```

Comment:
Could we get a comment on what this semaphore protects, please?  I don't mind the shorter name, but an indication that this mutexes the motor outputs or whatever would be good

Reply:
Fixed